Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -38,9 +39,54 @@
 FETCH_TIME = demisto.params().get('fetch_time', '3 days')
 FETCH_SIZE = int(demisto.params().get('fetch_size', 50))
 INSECURE = not demisto.params().get('insecure', False)
+TIME_METHOD = demisto.params().get('time_method', 'Simple-Date')
+
+# if timestamp than set the format to iso.
+if 'Timestamp' in TIME_METHOD:
+    TIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f'
+
+
+def get_timestamp_first_fetch(last_fetch):
+    """Gets the last fetch time as a datetime and converts it to the relavent timestamp format.
+
+    Args:
+        last_fetch(datetime): A datetime object setting up the last fetch time
+    """
+    if TIME_METHOD == 'Timestamp-Seconds':
+        return int(last_fetch.timestamp())
+
+    elif TIME_METHOD == 'Timestamp-Milliseconds':
+        return int(last_fetch.timestamp() * 1000)
```

Comment:
why Don't save the fetch as an int, it will be more accurate this way

Reply:
I didn't want to change the entire logic of the fetch. So in this function after getting the last fetch time I convert it to the correct timestamp format.

I can change the integration to work with a timestamp throughout the fetch process but I thought it would be better to be less intrusive and change the overall logic.